lib/widgets/gallery_widget.dart

```diff
@@ -84,30 +53,41 @@ class GalleryImage extends StatelessWidget {
   const GalleryImage({
     Key? key,
     required this.path,
-    required this.selected,
+    this.autofocus = false,
   }) : super(key: key);
 
   final String path;
-  final bool selected;
+  final bool autofocus;
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      color: selected ? Theme.of(context).focusColor : Colors.transparent,
-      child: Hero(
-        tag: path,
-        child: Center(
-          child: PhysicalModel(
-            color: Colors.black,
-            elevation: 10,
-            shadowColor: Colors.black,
-            child: Tooltip(
-              message: p.basename(path),
-              child: Image.file(File(path)),
+    return Focus(
+      autofocus: autofocus,
+      child: Builder(builder: (context) {
+        return GestureDetector(
+          onTap: () {
+            Focus.of(context).requestFocus();
+          },
+          child: Container(
+            padding: const EdgeInsets.all(8),
+            color: Focus.isAt(context) ? Theme.of(context).focusColor : Colors.transparent,
+            child: Hero(
+              tag: path,
+              child: Center(
+                child: PhysicalModel(
+                  color: Colors.black,
+                  elevation: 10,
+                  shadowColor: Colors.black,
+                  child: Tooltip(
+                    message: p.basename(path),
+                    child: Image.file(File(path)),
+                  ),
+                ),
+              ),
             ),
           ),
-        ),
-      ),
+        );
+      }),
     );
   }
```

Comment:
I think we can replace the whole thing with an `IconButton` as well, that has a focus node and focus feedback integrated